mesos_master/datadog_checks/mesos_master/mesos_master.py

```diff
@@ -237,6 +237,7 @@ def _check_leadership(self, url, tags=None):
 
         if state_metrics is not None:
             self.version = [int(i) for i in state_metrics['version'].split('.')]
+            self._collect_metadata(state_metrics)
```

Comment:
```suggestion
            self.set_metadata('version', state_metrics['version'])
```
I don't think it's worth abstracting away the submission of the version. It makes the code harder to read